packages/inspect-server/src/server.ts

```diff
@@ -17,6 +19,15 @@ export interface InspectServer {
   port: number;
 }
 
+function appDir() {
+  let index = require.resolve("@effection/inspect-ui/dist-app/index.html");
+  assert(existsSync(index), `the @effection/inspect-ui app has gone missing!
+Expected ${index} to exist, but it did not.
```

Comment:
I'm a little bit worried about breaking the encapsulation between packages here because it means that the server package cannot be re-organized freely without breaking packages that consume it. This is a minor annoyance for the inspect server since it is part of the same mono-repo and the tests should start failing if we change the location of the built app. However, this will also break dependent packages that want to serve the inspector app without necessarily using the inspect server. Is there a reason why this function cannot live in the `@effection/inspect-ui` package?